README.md

```diff
@@ -36,8 +36,11 @@ var Statful = require('statful-client');
 
 // Creates an object with the configuration and pass it to the client
 var config = {
-    token: 'MyAppToken',
     app: 'AccountService',
+    transport: 'api',
+    api: {
+        token: 'YOUR_TOKEN_FOR_STATFUL_API'
```

Comment:
Detail, this should be just `STATFUL_API_TOKEN` - no need for direct language in tech docs.